apps/explorer/src/components/owned-objects/views/ObjectFieldsCard.tsx

```diff
@@ -47,6 +48,13 @@ export function ObjectFieldsCard({ id }: ObjectFieldsProps) {
         functionName
     );
 
+    // Set the active field name to the first field in the struct on load
+    useEffect(() => {
```

Comment:
I think it'd be slightly better to put this in the `onSuccess` option of `useGetNormalizedMoveStruct` because setting the active field name is more a result of the query data being successfully returned versus being a side effect of rendering